### ISSUE TITLE
When creating a new session, after having created or attempting to create another, "Select Sprint" drop-down does not populate/activate.

### ISSUE THREAD

**Issue by jessicadcrl** [microsoft/azure-boards-estimate#332]
WHEN: User creates a session, then creates another session.
OR: User is configuring a new session, cancels, then opens window to re-try creating a new session.
THEN: After the Team drop-down is filled, the "Select Sprint" drop-down does not activate nor populate.

**Comment by jessicadcrl** [comment]
I am trying to upload a screenshot but I cannot

**Comment by AminTi** [comment]
Hi, Please drag and drop the screenshot here. Could you describe your environment? Also, could you check if there are any errors in the browser console or network log?

**Comment by AminTi** [comment]
I’ve been waiting on a screenshot and description of your environment. If you're still experiencing the same issue, please open a new ticket so we can address it accordingly.
